app/controllers/charges_controller.rb

```diff
@@ -5,26 +5,27 @@ def new
   end
 
   def create
-    @amount = (cart.total * 100)
+      @amount = (cart.total * 100)
 
-    customer = Stripe::Customer.create(
-      :email => 'example@stripe.com',
-      :card  => params[:stripeToken]
-    )
-
-    charge = Stripe::Charge.create(
-      :customer    => customer.id,
-      :amount      => @amount,
-      :description => 'Rails Stripe customer',
-      :currency    => 'usd'
-    )
-
-    order = Crusher.create_order(session, current_user, cart)
-    session[:donations].clear
-    if current_user.phone_number
-      SendNotification.new.text_message(current_user.phone_number, order, current_user)
-    end
-    redirect_to order_path(order)
+      customer = Stripe::Customer.create(
+        :email => 'example@stripe.com',
+        :card  => params[:stripeToken]
+      )
+
+      charge = Stripe::Charge.create(
+        :customer    => customer.id,
+        :amount      => @amount,
+        :description => 'Rails Stripe customer',
+        :currency    => 'usd'
```

Comment:
Unnecessary spacing detected.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.